starter-code/src/app/contact-list/contact-list.component.ts

```diff
@@ -8,7 +8,9 @@ import contactList from '../contacts';
 })
 export class ContactListComponent implements OnInit {
   contacts: Object[];
-  newContact: Object = {};
+  //newContact: Object = {};
+
+  newContact: Contact = new Contact();
```

Comment:
Some people prefer to not declare types in `new...` statements, since the type is obvious, and we're left with the more simple:
`newContact = new Contact();`

This is the benefit of Typescript with flexible typing